packages/shared/src/api/dto/user/create.user.dto.ts

```diff
@@ -0,0 +1,5 @@
+export interface CreateUserDto {
```

Comment:
move to
/home/michal/dev/priv/camp-trello-clone/packages/shared/src/api/command

and call: CreateUserCommand

we use dtos and commands to model our API